examples/SharedMemory/b3RobotSimulatorClientAPI_NoDirect.h

```diff
@@ -89,6 +89,55 @@ struct b3RobotSimulatorLoadSoftBodyArgs
 	}
 };
 
+
+struct b3RobotSimulatorLoadDeformableBodyArgs
```

Comment:
A lot of of parameters are not initialized in the constructor. Can you initialize them either with some reasonable value or -1 (or some flag) to mark that they should not be used?

Reply:
See for example this: https://github.com/bulletphysics/bullet3/blob/master/examples/SharedMemory/b3RobotSimulatorClientAPI_NoDirect.h#L100

Reply:
Done. I also tuned the parameter of the cloth a little bit.